### ISSUE TITLE
Have JSLint output the correct line numbers

### ISSUE THREAD

**Issue by OscarGodson** [OscarGodson/EpicEditor#90]
I've noticed the line numbers are usually way off. It'd be nice if it actually matched up.

**Comment by johnmdonahue** [comment]
The problem here is that we can't run JSHint on stand-alone non-catted files. Basically we're running JSHint only against EE core meaning a separate temp build and lint with Marked skipped. To get line numbers to match, we would need to be able to lint the entire ee.js and relax `.jshintrc` to get Marked to pass on the full build.

Another option would be to move Marked out as a standalone module and either just pollute the global or pass it EE to extend as a parser which seems like the direction you're wanting to go anyway. If we could do this then the whole `intro.js` hack would be unnecessary and line number matching would be much simpler.

See also #89.

**Comment by OscarGodson** [comment]
Hmm, I like moving Marked out... I'd like to decouple as much as possible anyway. Maybe we should leave this for when we do the modules. I just stubbed out these tickets. Not important, just nice to haves.

We can just say blocked on #33.

**Comment by johnmdonahue** [comment]
Cool yeah this makes since to lump in with #33. As a side note the line numbers should only be off by 6, or the # of lines in the `intro.js` file. Is that what you're seeing? The rest of the code is in `editor.js` and those are the only 2 files catted. It should never line up to the line numbers in the built file since that's not technically were the error resides, it's in the src. Regardless this will be a trivial fix when we can pull marked out as an extension.

**Comment by johnmdonahue** [comment]
@OscarGodson I think #114 takes care of this since `intro.js` has been removed. Also, I realize we don't yet have a full on extension interface but #114 also takes care of at least part of #33. Just wanting to tie up some loose ends.
